source/templates/writing-helpers.md

```diff
@@ -218,24 +218,44 @@ And two named arguments:
 * `format="YYY MM DD h:mm"`
 * `locale="en"`
 
-### Stateful Helpers
+### Service Helpers
```

Comment:
Please do not name these "Service Helpers", that is not a thing. "Helpers with Services" or "Stateful Heplers" or "Class-based Helpers" would all be more accurate.

I'd not like the name "Service Helper" to be something we encourage people to use throughout the community and ecosystem.